ImageGallery/src/org/sleuthkit/autopsy/imagegallery/datamodel/DrawableDB.java

```diff
@@ -174,11 +174,14 @@
 
     /**
      * Enum to track Image gallery db rebuild status for a data source
+     * 
+     * DO NOT add in the middle.
      */
     public enum DrawableDbBuildStatusEnum {
-        UNKNOWN, /// no known status
-        IN_PROGRESS, /// drawable db rebuild has been started for the data source
-        COMPLETE;       /// drawable db rebuild is complete for the data source
+        UNKNOWN,        /// drawable db does not have an entry for this data source
+        IN_PROGRESS,    /// drawable db rebuild has been started for the data source
+        COMPLETE,       /// drawable db rebuild is complete for the data source
+        DEFAULT;        /// drawable db needs to be updated for this data source
```

Comment:
We should make these not just about rebuild.  
IN_PROGRESS: Ingest or rebuild is in progress for the data source
COMPLETE: All files in the data source have had file type detected
DEFAULT: Not all files in the data source have had file type detected